Purchases/Public/RCPurchasesErrorUtils.m

```diff
@@ -115,6 +117,8 @@
             return @"INVALID_APPLE_SUBSCRIPTION_KEY";
         case RCIneligibleError:
             return @"INVALID_OFFER";
+        case RCBackendInvalidSubscriberAttributesError:
+            return @"SUBSCRIBER_ATTRIBUTES_SYNC_ERROR";
```

Comment:
I normally make these match the error code, so I would make it @"INVALID_SUBSCRIBER_ATTRIBUTES"